wallet/src/actors/worker/methods.rs

```diff
@@ -1153,3 +1195,10 @@ impl Worker {
         wallet.export_master_key(password).map_err(Error::from)
     }
 }
+
+fn is_old_epoch(birth_date: u32, checkpoint_zero_timestamp: i64, checkpoints_period: u16) -> bool {
```

Comment:
I don't understand the name of this function, what epoch should be old? It checks if the birth date epoch is greater than the current epoch. Maybe rename it to `validate_birth_date` and return a `Result<()>` using this line in case of error:

    if total_epochs < birth_date {
        return Err(Error::InvalidBirthDate(epoch));
    }